lib/puppet/ssl/host.rb

```diff
@@ -382,6 +368,33 @@ def state
       return 'revoked'
     end
   end
+
+  private
```

Comment:
I think this makes this officially a backwards-incompatible change. Intentional?